src/components/home/create/CheckProcess.tsx

```diff
@@ -0,0 +1,19 @@
+import { selectInProcess } from '@store/slices/createChallengeSlice';
+import { useSelector } from 'react-redux';
+import { Navigate } from 'react-router-dom';
+
+interface CheckProcessProps {
+  children: JSX.Element;
+  currentStep: number;
+}
+
+const CheckProcess = ({ children, currentStep }: CheckProcessProps) => {
+  const inProcess = useSelector(selectInProcess);
```

Comment:
```suggestion
  const isInProcess = useAppSelector(selectInProcess);
```
* boolean 변수임을 변수명을 통해 구분해 주셨으면 좋겠습니다.
* 명확한 type 적용을 위해 useAppSelector, useAppDispatch를 사용할 수 있도록 주의해주세요.